docker-compose.nodeodm.gpu.nvidia.yml

```diff
@@ -11,6 +11,7 @@ services:
       - WO_DEFAULT_NODES
   node-odm:
     image: opendronemap/nodeodm:gpu
+    container_name: node-odm-1
```

Comment:
Did you test with:

`./webodm.sh start --default-nodes 3` ?

Reply:
My apologies. I forgot there would be conflict here. 

I still don't fully understand the way to the naming. Can you give me some advice?

Reply:
You need to understand how docker-compose works.

This is a good start: https://docs.docker.com/compose/

Read it all.